students/yixingxu/lesson02/Fizz-Buzz.py

```diff
@@ -0,0 +1,9 @@
+for i in range(1,100):
+	if i%3 ==0 and i%5 ==0:
+		print('FizzBuzz')
+	elif i%3 ==0:
+		print('Fizz')
+	elif i%5 ==0:
+		print('Buzz')
+	else: print(i)
+		
```

Comment:
your code is over-indented; per pep8 guidelines indent should be exactly 4 spaces (not tabs).

you should be able to configure whichever editor you are using to indent correctly for you.